buildconfig/Jenkins/systemtests

```diff
@@ -98,11 +98,16 @@ echo "CheckMantidVersion.OnStartup = 0" >> $userprops
 default_save_directory=${WORKSPACE}/build/Testing/SystemTests/scripts/
 find ${default_save_directory} -name "*-mismatch*" -delete
 
-# Run
-NTHREADS=${BUILD_THREADS:?}
+# Run. Use SYSTEST_NPROCS over BUILD_THREADS if defined
```

Comment:
Would `SYSTEST_NPROCS=${SYSTEST_NPROCS:-$BUILD_THREADS}` not be a simpler way to express this?

Then `SYSTEST_NPROCS` would be used in the call to run the tests.

Reply:
I still need to check BUILD_THREADS is defined though...
```
BUILD_THREADS=${BUILD_THREADS:-1}
SYSTEST_NPROCS=${SYSTEST_NPROCS:-$BUILD_THREADS}
```
I'll get rid of the hard limit and we can set them per node.

Reply:
Oh, I thought we defaulted `BUILD_THREADS` if it was not defined.